src/org/labkey/test/pages/study/CreateNewVisitPage.java

```diff
@@ -0,0 +1,98 @@
+package org.labkey.test.pages.study;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Checkbox;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.SelectWrapper;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class CreateNewVisitPage extends LabKeyPage<CreateNewVisitPage.ElementCache>
+{
+    public CreateNewVisitPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static CreateNewVisitPage beginAt(WebDriverWrapper webDriverWrapper)
+    {
+        return beginAt(webDriverWrapper, webDriverWrapper.getCurrentContainerPath());
+    }
+
+    public static CreateNewVisitPage beginAt(WebDriverWrapper webDriverWrapper, String containerPath)
+    {
+        webDriverWrapper.beginAt(WebTestHelper.buildURL("study", containerPath, "createVisit"));
+        return new CreateNewVisitPage(webDriverWrapper.getDriver());
+    }
+
```

Comment:
It doesn't look like this page really needs one, but want to check and make sure you considered if there is a need for a `waitForPage` check to make sure the page is loaded.

Reply:
Never needed it.